package.json

```diff
@@ -75,7 +75,7 @@
     "jest": "28.0.3",
     "jest-environment-jsdom": "^28.0.2",
     "jest-fail-on-console": "^2.4.2",
-    "jscodeshift": "^0.11.0",
+    "jscodeshift": "^17.0.0",
```

Comment:
In version `0.11.0` it throws an error when parsing code with type imports inside named imports, like the following:
```typescript
import { type ReactElement, useEffect, useState } from 'react';
```

Reply:
thanks